pkg/util/cgroups/cgroupv2_pids.go

```diff
@@ -21,13 +21,13 @@ func (c *cgroupV2) GetPIDStats(stats *PIDStats) error {
 
 	stats.PIDs = nil
 	if err := parseFile(c.fr, c.pathFor("cgroup.procs"), func(s string) error {
-		pid, err := strconv.ParseInt(s, 10, 64)
+		pid, err := strconv.Atoi(s)
```

Comment:
https://github.com/stephengroat/datadog-agent/security/code-scanning/270?query=ref%3Arefs%2Fpull%2F1%2Fmerge

Reply:
(same as for pkg/util/cgroups/cgroupv1_pids.go)